src/storage/src/source/reclock.rs

```diff
@@ -370,9 +370,13 @@ impl ReclockOperator {
             let compare_and_append_result = self
                 .write_handle
                 .compare_and_append(
-                    new_bindings
-                        .iter()
-                        .map(|(partition, diff)| ((&(), *partition), &new_ts, diff)),
+                    new_bindings.iter().map(|(partition, diff)| {
+                        let diff: i64 = diff
```

Comment:
Can we make the binding collection have a `u64` diff instead?

Reply:
`u64` doesnt implement `Abelian`?